app/controllers/api/v1/books_controller.rb

```diff
@@ -0,0 +1,15 @@
+module Api
+  module V1
+    class BooksController < ApplicationController
+      def index
+        render_paginated Book.includes(:rents).all
+      end
+
+      def show
+        render json: Book.includes(:rents).find(params[:id])
+      rescue StandardError => exception
```

Comment:
If I record correctly, this is not  required, the `rescue_from` directive  will automatically catch the error

Reply:
Great! Done.